.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
These personal settings of you IDE do not belong in the project repo.

Please remove this file. And maybe also add an exclusion rule to your global .gitignore file. We prefer not to add rules for custom tools to the project's .gitignore.

@sgoll If you agree, of course ;)

Reply:
Okay, I was unsure if I should add them or not. I will just keep them on my machine.